### PR TITLE
Tec 788

### PR DESCRIPTION
**Pull request by keithamus**
This hides the Subscribe links when the user is already a subscriber.

Goes most of the way to implementing TEC-788 but feeblags needs some work.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -242,7 +242,14 @@ export default class Navigation extends React.Component {
 
   render() {
     const { searching } = this.state;
+    const { userIsSubscriber } = this.props;
+    let { accordionData } = this.props;
     const svgUri = { uri: this.props.svgUri } || {};
+    if (userIsSubscriber) {
+      accordionData = accordionData.filter((accordianLink) => (
```
Comment (gcedo) [inline src/index.js]: typo?

**Comment by Ericat** [comment]
@keithamus all tests pass, it looks good to me. All there is left to do is edit the menu links in fe-blogs, right?

**Comment by keithamus** [comment]
@Ericat correct. Well, that and updating the package.json/shrinkwrap to the new version.

**Comment by Ericat** [comment]
ok, do you want me to merge or shall @gcedo go ahead with it?

**Comment by keithamus** [comment]
I'll that one up to you :wink:

**Comment by gcedo** [comment]
@Ericat LGTM, go nuts.

**Pull request merged**